pages/tools.md

```diff
@@ -43,6 +43,7 @@ development process:
 * [pa11y](http://pa11y.org/) and is like a11y, but consists of a larger suite of tools, including command-line and JavaScript APIs, a [web service](https://github.com/nature/pa11y-webservice) and a [dashboard](https://github.com/nature/pa11y-dashboard) for monitoring accessibility reports across multiple sites.
 * [ra11y](https://github.com/benbalter/ra11y) is a Ruby-based accessibility testing tool tuned for use with [Jekyll](http://jekyllrb.com/) and static sites.
 * [webalin](http://webalin.readthedocs.org/en/latest/) is a Python-based 508 compliance linter for HTML.
+* [AccessLintCI](https://github.com/accesslint/accesslint-ci) offers accessibility auditing via CircleCI builds. More information on how to add this to an 18F site [here](https://github.com/18F/development-guide/tree/master/accessibility_scanning). More specific questions can be directed to the AccessLint team on their [slack channel](accesslint.slack.com).
```

Comment:
>More specific questions can be directed to the AccessLint team on their [slack channel](accesslint.slack.com). 

Maybe leave this out for now since people can submit issues via GitHub, and any additional contact info like this _should_ be provided by them on their repo or website.